package.json

```diff
@@ -1,6 +0,0 @@
-{
-  "dependencies": {
-    "postcss-loader": "^7.0.0",
-    "sass": "^1.52.3"
-  }
-}
```

Comment:
Instead of making a separate commit for this, I would instead squash this to the original commit to keep history cleaner by keeping "noise" out :)

Reply:
Kuitti